src/Nordea/Components/DropdownOptions.elm

```diff
@@ -0,0 +1,252 @@
+            in
+            Html.div
+                [ css
+                    [ displayFlex
+                    , flexDirection column
+                    , position absolute
+                    , top (pct 100)
+                    , left (rem 0)
+                    , right (rem 0)
+                    , if dropdown.hasFocus then
+                        display block
+
+                      else
+                        display none
+                    ]
+                ]
+                [ Html.ul
+                    [ css
+                        [ overflowY scroll
+                        , maxHeight (rem 16.75)
+                        , dropdownStyles
+                        , listStyle none
+                        ]
+                    ]
+                    (dropdown.options |> List.map viewOption)
+                ]
+    in
+    Html.column attrs
+        [ Html.row [ css [ displayFlex, justifyContent spaceBetween, marginBottom (rem 0.2) ] ]
+            [ Common.viewLabel { label = dropdown.label, isError = False } []
+            , dropdown.requirednessHint |> Html.viewMaybe RequirednessHint.view
+            ]
+        , Html.div
+            [ class "nordea-select"
+            , Events.on "focusout" (Decode.succeed (dropdown.onFocus False))
+            , Events.on "focusin" (Decode.succeed (dropdown.onFocus True))
+            , css [ position relative ]
+            , tabindex -1
+            ]
+            [ Html.select
+                [ css [ display none ] ]
+                (dropdown.options |> List.map (\option -> Html.option [ name option.name ] [ Html.text option.label ]))
+            , Html.div
+                [ class "select-selected"
+                , tabindex -1
+                , css
+                    [ height (rem 3)
+                    , width (pct 100)
+                    , backgroundColor transparent
+                    , padding4 (rem 0.5) (rem 1) (rem 0.5) (rem 1)
+                    , border3 (rem 0.0625) solid Colors.grayMedium
+                    , if dropdown.hasFocus then
+                        borderRadius4 (rem 0.25) (rem 0.25) (rem 0.0) (rem 0.0)
+
+                      else
+                        borderRadius4 (rem 0.25) (rem 0.25) (rem 0.25) (rem 0.25)
+                    , fontSize (rem 1.0)
+                    , lineHeight (rem 1.4)
+                    , color inherit
+                    , cursor pointer
+                    , displayFlex
+                    , alignItems center
+                    , justifyContent spaceBetween
+                    , focus
+                        [ Css.property "box-shadow" ("0rem 0rem 0rem 0.0625rem " ++ Themes.colorVariable Themes.SecondaryColor Colors.blueNordea)
+                        , outline none
+                        ]
+                    , Css.property "appearance" "none"
+                    , Css.property "-moz-appearance" "none"
+                    , Css.property "-webkit-appearance" "none"
```

Comment:
Looks to be supported by all major browsers without the prefix: https://caniuse.com/mdn-css_properties_appearance_none
```suggestion
```